ci-operator/step-registry/cucushift/installer/check/aws/custom-vpc/cucushift-installer-check-aws-custom-vpc-commands.sh

```
@@ -71,9 +71,20 @@ jq --arg k $expect_k --arg v $expect_v -r '[.Subnets[] | select(any(.Tags[]; .Ke
 echo "--------------------------------"
 
 cnt=$(jq --arg k $expect_k --arg v $expect_v -r '[.Subnets[] | select(any(.Tags[]; .Key == $k and .Value == $v))] | length' $out)
-expect_cnt=$(echo ${ic_subnets} | wc -w)
+
+# Calculate expected count based on whether Edge Zones are enabled
+if [[ ${ENABLE_AWS_EDGE_ZONE:-no} == "yes" ]]; then
+  # For Edge Zone scenarios, exclude subnets tagged as 'kubernetes.io/cluster/unmanaged:true'
+  unmanaged_cnt=$(jq -r '[.Subnets[] | select(any(.Tags[]; .Key == "kubernetes.io/cluster/unmanaged" and .Value == "true"))] | length' $out)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)